Source/Core/VideoCommon/VertexShaderManager.cpp

```diff
@@ -417,6 +419,21 @@ void VertexShaderManager::SetConstants()
 			g_fProjectionMatrix[14] = -1.0f;
 			g_fProjectionMatrix[15] = 0.0f;
 
+			// Heuristic to detect if a GameCube game is in 16:9 anamorphic widescreen mode.
+			if (!SConfig::GetInstance().bWii)
+			{
+				// Due to the BT.601 standard which the GameCube is based on being a compromise 
+				// between PAL and NTSC, neither standard gets square pixels. They are each off 
+				// by ~9% in opposite directions.
+				// Just in case any game decides to take this into account, we do these tests
+				// with a large amount of slop.
+				float aspect = fabsf(rawProjection[2] / rawProjection[0]);
+				if (fabsf(aspect - 16.0f/9.0f) < 16.0f/9.0f * 0.11) // within 11% of 16:9
```

Comment:
I personally prefer spaces around the ```/```; but what do you think of making those two name constants?